packages/utilities/src/shadowDom/contexts/MergeStylesShadowRootContext.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
+import { GLOBAL_STYLESHEET_KEY } from '@fluentui/merge-styles';
+import { useMergeStylesHooks } from '../hooks/useMergeStylesHooks';
+
+export type MergeStylesShadowRootContextValue = {
+  /**
+   * Map of stylesheets available in the context.
+   */
+  stylesheets: Map<string, CSSStyleSheet>;
+
+  /**
+   * Shadow root for this context.
+   */
+  shadowRoot?: ShadowRoot | null;
```

Comment:
Is the `ShadowRoot` type implicit here? I don't see an import with it...

Reply:
[`ShadowRoot` is from TS's DOM lib types](https://github.com/microsoft/TypeScript/blob/main/src/lib/dom.generated.d.ts#L21376).